lib/dialects/postgres/query-generator.js

```diff
@@ -789,9 +791,9 @@ const QueryGenerator = {
     return sql;
   },
 
-  pgEnumDrop(tableName, attr, enumName) {
+  pgEnumDrop(tableName, attr, enumName, cascade) {
     enumName = enumName || this.pgEnumName(tableName, attr);
-    return 'DROP TYPE IF EXISTS ' + enumName + '; ';
+    return 'DROP TYPE IF EXISTS ' + enumName + (cascade ? ' CASCADE' : ' RESTRICT') + '; ';
```

Comment:
@janmeier It looks like they added that `RESTRICT` you were asking for, but then no new activity before stalebot closed it.  Does this look good to you now?